libraries/SITL/SIM_Blimp.cpp

```diff
@@ -143,6 +197,12 @@ void Blimp::calculate_forces(const struct sitl_input &input, Vector3f &body_acc,
 
 #if 0 //"Wobble" attempt
   rot_T.y = fin[0].Fz * radius + fin[1].Fz * radius;
+  // @LoggerMessage: SRT2
+  // @Description: Transformed Simulated Blimp Rotational forces
+  // @Field: TimeUS: Time since system startup
+  // @Field: rtx: zero
+  // @Field: rty: zero
```

Comment:
_rty_ is the wobble rotational forces - set in the line right above this addition.

Reply:
Oops!  Fixed, thanks!